data/weapons.txt

```diff
@@ -130,8 +130,8 @@ outfit "Modified Blaster Turret"
 		"hit effect" "blaster impact"
 		"inaccuracy" 4
 		"turret turn" 4
-		"velocity" 7.5
-		"lifetime" 60
+		"velocity" 15
+		"lifetime" 30
```

Comment:
The modified blaster turret has a higher velocity than the modified blaster gun?